spec/api_spec.rb

```diff
@@ -21,10 +21,10 @@ class Bar; end
       end
 
       context "when setting custom options" do
-        before { subject.setup foo: { bar: "baz" }, url: "https://api.example.com" }
+        before { subject.setup foo: { bar: "baz" }, url: "https://api.example.com", timeout: 10, open_timeout: 2 }
```

Comment:
Haven't played around w/ the faraday [retry options](https://github.com/lostisland/faraday/blob/master/lib/faraday/request/retry.rb#L25) much. Is there anything there you could take advantage of? 

Regardless it sounds like this is a separate issue and there's a separate [PR](https://github.com/remiprev/her/pull/452) explaining how to use `timeout` and `open_timeout`. I'll go ahead and close this PR then.